skada/_reweight.py

```diff
@@ -7,6 +7,7 @@
 import warnings
 
 import numpy as np
+from cvxopt import matrix, solvers
```

Comment:
why not using SciPy minimizer ?

Reply:
The problem TarS solves is a quadratic problem under constraints. I don't think SciPy can solve this out of the box and CVXOPT is made for this .

Reply:
Look at #94, the KMM method solves also a quadratic problem under constraints but with SciPy. But tbh I havent yet looked at the TarS paper thoroughly, so maybe I'm missing something. Also if you think that CVXOPT is better for this kind of problem, yeah sure go ahead

Reply:
If it is possible it could be nice to be consistent in the toolbox and not to add a new requirement